pkg/cluster/cluster_config.go

```diff
@@ -58,7 +60,9 @@ func GetClusterServiceVersion(ctx context.Context, c client.Client, watchNameSpa
 		}
 	}
 
-	return nil, nil
+	return nil, kerrors.NewNotFound(
+		schema.GroupResource{Group: gvk.ClusterServiceVersion.Group},
+		"DataScienceCluster")
```

Comment:
why the name is `DataScienceCluster` ? i think `DataScienceCluster` was only one of the owner set in the CSV

Reply:
> why the name is `DataScienceCluster` ? i think `DataScienceCluster` was only one of the owner set in the CSV

Reasonable. Since the request is not for particular name and only error type matters I can leave it blank or put Kind (ClusterServiceVersion) there. What do you think?

Reply:
i dont have strong opinion on this, Kind could be better than a blank for debug reason.

Reply:
Updated